GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day2/InheritanceI.java

```diff
@@ -0,0 +1,32 @@
+package hackkerank.day2;
```

Comment:
According to Java standards and common practices, we should declare every class in its own source file.

https://dzone.com/articles/why-single-java-source-file-can-not-have-more-than